package/kedro_viz/server.py

```diff
@@ -32,7 +31,7 @@ def populate_data(
     data_access_manager: DataAccessManager,
     catalog: DataCatalog,
     pipelines: Dict[str, Pipeline],
-    session_store: BaseSessionStore,
+    session_store: Optional[BaseSessionStore],
```

Comment:
Sounds like a minor comment, but it's actually important... why is this `Optional`?

Reply:
My bad I was getting a weird test_cli.error earlier so I thought this might be the issue but I think the issue finally was a session-scoped pytest fixture.